<?xml version="1.0"?>

<launch>
<node pkg="erc_supervisor" type="erc_supervisor" name="erc_supervisor" output="screen">
<rosparam command="load" file="$(find erc_supervisor)/cfg/ErcSupervisor.yaml"/>
</node>
</launch>